bash_completions.go

```diff
@@ -160,7 +160,7 @@ __%[1]s_handle_filename_extension_flag()
 __%[1]s_handle_subdirs_in_dir_flag()
 {
     local dir="$1"
-    pushd "${dir}" >/dev/null 2>&1 && _filedir -d && popd >/dev/null 2>&1
+    pushd "${dir}" >/dev/null 2>&1 && _filedir -d && popd >/dev/null 2>&1 || exit
```

Comment:
Completions are run as functions in the current shell. If `popd` or `pushd` fail, `|| exit` will exit the entire shell, possibly closing your terminal. You need to use `|| return` instead

Reply:
changed to `return`